bigbluebutton-html5/imports/api/audio/client/bridge/FullAudioBridge.js

```diff
@@ -304,14 +314,14 @@ export default class FullAudioBridge extends BaseAudioBridge {
       const callerIdName = [
         `${this.userId}_${getAudioSessionNumber()}`,
         'bbbID',
-        isListenOnly ? `${GLOBAL_AUDIO_PREFIX}${this.voiceBridge}` : this.name,
+        isListenOnly ? `${GLOBAL_AUDIO_PREFIX}` : this.name,
```

Comment:
Is `${this.voiceBridge}` part of `getAudioSessionNumber()` now?

Reply:
voiceBridge doesn't exist anymore in that class. The rest of the callerId is filled server-side (bbb-webrtc-sfu) where the information comes from a trusted source (bbb-web).